src/models/SQLCalls.java

```diff
@@ -193,32 +193,44 @@ public void leaveRating(String locationID, String userID, String elevatorRating,
 // --------------------------------- For Profile.java ---------------------------------
 	
 	// sets a user's city
-	public void setCity(String city, String userID) {
+	public String setCity(String username, String city) {
```

Comment:
I changed the parameters of the profile methods, so the frontend will need usernames, not userIDs, to make changes to User table.